src/js/fs.js

```diff
@@ -263,26 +255,25 @@ fs.readFileSync = function(path) {
   return Buffer.concat(buffers);
 };
 
+fs.writeFile = function(path, outdata, callback) {
+  var data = ensureBuffer(outdata);
```

Comment:
Ok, I think I understand, so I made a coertion to string for all other
types except Buffer (Since we dont have Uint8Array).